### PR TITLE
fix: payment data filled event logs for few payment methods

### PR DESCRIPTION
**Pull request by vsrivatsa-juspay**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Added payment data filled event logs which was missing for few payment methods

## Checklist

- [ ] I ran `npm run re:build`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/Payments/PayPal.res [hunk 1]
```diff
@@ -48,6 +48,7 @@ let make = () => {
     ->then(result => {
       let result = result->JSON.Decode.bool->Option.getOr(false)
       if result {
+        UtilityHooks.useHandlePostMessages(~complete=true, ~empty=true, ~paymentType="paypal")
```
Comment (vsrivatsa-juspay) [inline src/Payments/PayPal.res:51]:
```suggestion
        UtilityHooks.useHandlePostMessages(~complete=true, ~empty=false, ~paymentType="paypal")
```

File: src/Payments/PaypalSDKHelpers.res [hunk 2]
```diff
@@ -30,6 +30,7 @@ let loadPaypalSDK = (
   ->then(result => {
     let result = result->JSON.Decode.bool->Option.getOr(false)
     if result {
+      UtilityHooks.useHandlePostMessages(~complete=true, ~empty=true, ~paymentType="paypal")
```
Comment (vsrivatsa-juspay) [inline src/Payments/PaypalSDKHelpers.res:33]:
```suggestion
      UtilityHooks.useHandlePostMessages(~complete=true, ~empty=false, ~paymentType="paypal")
```

**Review by PritishBudhiraja — DISMISSED** [review]
Looks good 👍🏼

**Review by PritishBudhiraja — COMMENTED** [review]
Looks good 👍🏼

**Review by PritishBudhiraja — DISMISSED** [review]
Looks good 👍🏼

**Review by PritishBudhiraja — DISMISSED** [review]

**Review by PritishBudhiraja — APPROVED** [review]

**Pull request merged**